src/MongoMemoryReplSet.ts

```diff
@@ -95,6 +103,7 @@ export default class MongoMemoryReplSet extends events.EventEmitter {
     // this function is only async for consistency with MongoMemoryServer
     // I don't see much point to either of them being async but don't
     // care enough to change it and introduce a breaking change.
+    // TODO : I pretty much agree on this
```

Comment:
Like @jloveridge suggested before should we keep this function async event if it is not returning any promise ?

Reply:
```js
async function() {
  return '123';
}
```

`async function` returns `Promise`, even in the function body was returned the string. It wraps the result in Promise implicitly.

Reply:
@nodkz Yes of course I understand that an async function always returns a Promise implicitly but in this case why wouldn't we just declare this function like this without the async keyword ? 
```javascript
function getDbName(): string {
  return this.opts.replSet.dbName;
}

Reply:
Because in `MongoMemoryServer.ts` we have the method:
```js
  async getDbName(): Promise<string> {
    const { dbName }: MongoInstanceDataT = await this.getInstanceData();
    return dbName;
  }
```

And for consistency better to leave it as Promise. I think that in future `MongoMemoryReplSet` will be rewritten from scratch.

Async `MongoMemoryServer.getDbName` allows to kill instance and when calling this method, you will get a new instance with a new DB name.